resources/code-snippets.md

```diff
@@ -56,7 +56,7 @@ cd moonbeam
 
 ## Development Accounts
 
---8<-- 'text/setting-up-local/dev-accounts.md'
+--8<-- 'text/setting-up-node/dev-accounts.md'
```

Comment:
for the text snippets, it looks like the directory is still "setting-up-local" instead of "setting-up-node" so this snippet is not rendering on the page